### DISCUSSION TITLE
63 版本样式未正常打包

### DISCUSSION THREAD

**Discussion by djknight1** [alibaba/formily#1563] [Q&A 2.x] [answered]
## 复现步骤：
项目 formily 升级至 63版本
webpack module 如下设置
![image](https://user-images.githubusercontent.com/33688921/121458062-4524ea00-c9db-11eb-81d7-1b9739ad7014.png)

## 当前表现
![image](https://user-images.githubusercontent.com/33688921/121458119-61c12200-c9db-11eb-8de4-798059159d61.png)

## 怀疑问题
63 版本增加了 effect，用户未手动引入样式文件的情况下，其是否会被 tree-shaking 掉？

**Comment by liuweiGL** [comment]
哭了，next 是scss？

**Reply by janryWang** [reply]
对，走的scss

**Comment by liuweiGL** [comment]
你先修改一下 node_modules 下 @formily/next:
```ts
sideEffects: [
// ...
*.less, // 删除
*.scss // 新增
]

```

**Comment by liuweiGL** [comment, marked as answer]
我现在提个 pr，sorry~

**Reply by liuweiGL** [reply]
#1564
